playbooks/roles/ocp-customization/tasks/main.yaml

```diff
@@ -1,10 +1,44 @@
----
 # tasks file for ocp4 post install customizations
+- name: Install openshift python module
+  pip:
+    executable: pip3
+    name: openshift
 
-- name: Configure RHCOS kernel options via MachineConfig
-  when: rhcos_kernel_options | length > 0
-  import_tasks: kernel.yaml
+- name:
+  block:
+    - name: Pause reboot node (Machineconfig)
```

Comment:
Why do we need to pause reboot node here? Will node reboot after install is completed?

Reply:
There are currently 2 customizations (maybe more in the future) that reboot nodes via machineconfig changes: kernel parameters, and sysctl.
I notice that the 2 first nodes to reboot (one master, one worker) have to reboot twice when both systcl parameters and kernel parameters are triggered. I think it is because they restart as soon as the kernel modification start, but in the meantime, sysctl has also changed and required another reboot. 

I want to implement what is described here (https://openshift.tips/machine-config/ section  Disable auto rebooting after a change with the machine-config-operator).
Disabling automatic reboot, apply all the changes, then reactivate reboot.